docs/how-to/store-large-files.md

```diff
@@ -29,3 +29,23 @@ A 1MiB sector is 1,048,576 bytes. Next, we multiply that by 254/256. Result: a 1
 | 8MiB        | 8MB (8,000,000 bytes)       | 8,323,072 bytes       |
 | 512MiB      | 512MB (512,000,000 bytes)   | 532,676,608 bytes     |
 | 32GiB       | 32GB (32,000,000,000 bytes) | 34,091,302,912 bytes  |
+
+### Storage deals with offline data transfer
+Filecoin's offline data transfer feature is recommended for petabyte-scale datasets and larger. This allows users with very large datasets to complete the data transfer step offline (e.g. by shipping hard drives from the client to the storage miner), while the storage deal continues to work as intended on-chain.
+
+It is implemented via a flag on the storage deal command that tells the client not to transfer the data over the network, and provides a piece CID (a unique identifier describing the data) to the miner instead, which a miner must then match for the deal to go through. This gives the client node flexibility in how it can set up the deal — for example, passing miners a specific location on a hard drive for the data they can use to generate the piece CID. 
+
+How to make storage deals with offline data transfer:
+1. Generate the unique pieceCID. (TODO include steps or links.)
```

Comment:
This TODO is easily resolved. The steps are:

1) Use the Lotus client to generate a CAR file of the input:

`lotus client generate-car <inputPath> <outputPath>`.

2) Use the Lotus client to generate the piece CID:

`lotus client commP <inputCarFilePath> <minerAddress>`.

The miner address is only needed in Step 2 because the proof type used depends on the miner's sector size.